src/features/collections/hooks.ts

```diff
@@ -104,7 +104,7 @@ export const useProcessStatePolling = <
   result: R,
   processStateKeys: StateKey[],
   options?: { baseInterval?: number; rate?: number; skipPoll?: boolean }
-) => {
+): Partial<{ [processStateKey in StateKey]: ProcessState }> => {
```

Comment:
Adding a return from useProcessStatePolling hook for easier/clearer access to polling states